docs/src/main/asciidoc/application-configuration-guide.adoc

```diff
@@ -158,6 +158,19 @@ You can add the `greeting.suffix`, remove the other properties, change the value
 As usual, the application can be packaged using `./mvnw clean package` and executed using the `-runner.jar` file.
 You can also generate the native executable with `./mvnw clean package -Pnative`.
 
+== Programmatically access the configuration
+
+You can access the configuration programmatically,
```

Comment:
Maybe split it into 2 separate sentences.

Reply:
done